### PR TITLE
CHEAP-913 feat(vendor): Add new mutations for approval process.

### PR DESCRIPTION
**Pull request by junzhengca**

### CODE REVIEW

File: src/app/controllers/VendorController.js [hunk 1]
```diff
@@ -13,6 +15,42 @@ class VendorController {
 
     // ADD MUTATION METHODS BELOW
 
+    updateVendorApprovalStatus(id, is_approved){
```
Comment (bholagabbar) [inline src/app/controllers/VendorController.js:24]: @junthehacker Add JS docs

**Review by bholagabbar — APPROVED** [review]
Approving but don't forget to add JS Docs

**Comment by junzhengca** [comment]
> Approving but don't forget to add JS Docs

Just added

**Review by ralph-dev — APPROVED** [review]
LGTM but ADD TYPES to JSDOCS!
@param {string} id ID of Vendor etc

**Pull request merged**